qa/rpc-tests/walletbackup.py

```diff
@@ -238,8 +238,21 @@ def run_test_with_scversion(self, scversion, ceasable = True):
             try:
                 #Create proof for WCert
                 quality = 1 if ceasable else 0
-                proof = mcTest.create_test_proof(sc_name, scid_swapped, epoch_number, quality, MBTR_SC_FEE,
-                    FT_SC_FEE, epoch_cum_tree_hash, constant, [addr_node1, addr_node1], [bwt_amount1, bwt_amount2])
+
+                if scversion < 2:
+                    prev_cert_hash = None
+
+                proof = mcTest.create_test_proof(sc_name,
+                                                 scid_swapped,
+                                                 epoch_number,
+                                                 quality,
+                                                 MBTR_SC_FEE,
+                                                 FT_SC_FEE,
+                                                 epoch_cum_tree_hash,
+                                                 prev_cert_hash,
```

Comment:
Is it possible to do as in [qa/rpc-tests/sc_cert_getblocktemplate.py]? This would also remove the if at lines 242-243
```suggestion
                                                 prev_cert_hash if scversion >=2 else None,
```

Reply:
Done in f1aa8bb237c592621a4f635666ee5a4761bbb988